products/views.py

```diff
@@ -45,3 +49,65 @@ class ProductColourListView(ListAPIView):
 class ProductSizeListView(ListAPIView):
     queryset = ProductSize.objects.all()
     serializer_class = ProductSizeListSerializer
+
+
+class ProductReviewListView(APIView):
```

Comment:
siz Listmas Add Review to Product degan API qilishingiz kerak. ya'ni productga review qushish imkonini beruvchi api